jina/clients/request/__init__.py

```diff
@@ -65,7 +65,7 @@ def request_generator(
                 endpoint=exec_endpoint, target=target_executor, parameters=parameters
             )
         else:
-            if not isinstance(data, Iterable):
+            if not isinstance(data, Iterable) or isinstance(data, Document):
```

Comment:
Is this needed because `Document` is now an iterable?

Reply:
yes exactly